src/electron-main.ts

```diff
@@ -149,7 +161,8 @@ async function loadConfig(): Promise<void> {
     const asarPath = await getAsarPath();
 
     try {
-        global.vectorConfig = loadJsonFile(asarPath, "config.json");
+        console.log(`Loading global config: ${path.join(asarPath, LocalConfigFilename)}`);
```

Comment:
I would say this is the 'app' config file really, rather than global, as it's just the one that ships with the app.

```suggestion
        console.log(`Loading app config: ${path.join(asarPath, LocalConfigFilename)}`);
```